metrics/sender/syslog.go

```diff
@@ -0,0 +1,35 @@
+package sender
+
+import (
+	"log/syslog"
+
+	"github.com/openfresh/plasma/config"
+	metrics "github.com/rcrowley/go-metrics"
+)
+
+const Syslog = "syslog"
```

Comment:
__[golint]__
* [golint] exported const Syslog should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->